tsdb/tsm1/file_store.go

```diff
@@ -339,22 +343,29 @@ func (t *fileTracker) AddBytes(bytes uint64, level int) {
 	atomic.AddUint64(&t.diskBytes, bytes)
 
 	labels := t.Labels()
+	if level > 5 {
+		level = 5
+	}
 	labels["level"] = fmt.Sprintf("%d", level)
 	t.metrics.DiskSize.With(labels).Add(float64(bytes))
 }
 
 // SetFileCount sets the number of files in the FileStore.
 func (t *fileTracker) SetFileCount(files map[int]uint64) {
 	labels := t.Labels()
-	level := uint64(0)
+	maxLevel := uint64(0)
 	for k, v := range files {
-		labels["level"] = fmt.Sprintf("%d", k)
-		if uint64(k) > level {
-			level = uint64(k)
+		level := uint64(k)
+		if level > 5 {
+			level = 5
+		}
+		labels["level"] = fmt.Sprintf("%d", level)
+		if level > maxLevel {
+			maxLevel = level
 		}
 		t.metrics.Files.With(labels).Set(float64(v))
 	}
-	atomic.StoreUint64(&t.levels, level)
+	atomic.StoreUint64(&t.levels, maxLevel)
```

Comment:
What is `t.levels` used for? It's never returned from anywhere? Is this a regression that's been introduced?

Reply:
Used to keep track of metrics to clear later, but we could equally just set all the levels we care about to 0.

Reply:
Yeah I don't understand what it's for. We should clear this up.